samples/nextjs/next.config.js

```diff
@@ -11,6 +11,9 @@ const disconnected = process.env.JSS_MODE === JSS_MODE_DISCONNECTED;
 
 const nextConfig = {
 
+  // Allow specifying a distinct distDir when concurrently running app in a container
+  distDir: process.env.NEXTJS_DIST_DIR || ".next",
```

Comment:
Switch to single quotes
```suggestion
  distDir: process.env.NEXTJS_DIST_DIR || '.next',
```